src/modules/authorizer/role/AUT_Roles_v1.sol

```diff
@@ -135,164 +340,246 @@ contract AUT_Roles_v1 is
         _grantRole(DEFAULT_ADMIN_ROLE, initialAdmin);
     }
 
-    //--------------------------------------------------------------------------
-    // Public functions
+    // ========================================================================
+    // Public Getter Functions
+
+    // ------------------------------------------------------------------------
+    // Getter -  Role Management
 
     /// @inheritdoc IAuthorizer_v1
-    function checkForRole(bytes32 role, address who)
-        external
-        view
-        virtual
-        returns (bool)
-    {
-        return hasRole(role, who);
+    function getAdminRole() public pure returns (bytes32) {
+        return DEFAULT_ADMIN_ROLE;
     }
 
+    // ------------------------------------------------------------------------
+    // Getter -  Authorization
+
     /// @inheritdoc IAuthorizer_v1
-    function generateRoleId(address module, bytes32 role)
+    function getPermissions(address target_, bytes4 selector_)
         public
```

Comment:
I can't find any place where this is used internally, so can be external? (May also be true for other functions, but maybe i also oversaw something where its used)